scala/lambdas/ingest-event-aggregator/src/main/scala/uk/gov/nationalarchives/eventaggregator/Generators.scala

```diff
@@ -0,0 +1,18 @@
+package uk.gov.nationalarchives.eventaggregator
```

Comment:
It can go in the utils package. I've moved it there and updated the other projects which are using `() => UUID.randomUUID` to use `Generators[IO].generateUuid` and any `() => Instant.now` to use `Generators[IO].generateInstant`